File: curso-python/javascript-exercicios/ex035.js
const prompt = require("prompt-sync")();

console.log("-=".repeat(20));
console.log("Analisador de Triângulos");
console.log("-=".repeat(20));

const r1 = parseFloat(prompt("Digite a primeira reta: "));
const r2 = parseFloat(prompt("Digite a segunda reta: "));
const r3 = parseFloat(prompt("Digite a terceira reta: "));

if ((r1 + r2 > r3) && (r1 + r3 > r2) && (r2 + r3 > r1)) {
    console.log("Os segmentos acima PODEM FORMAR um triângulo");
} else {
    console.log("Os segmentos acima NÃO PODEM FORMAR um triângulo");
}

File: curso-python/ex035.py
#desenvolva um programa que leia o comprimento da três retas e diga ao usuário se elas podem ou não formar triângulos

print("-="*20)
print("analisador de triângulos")
print("-="*20)

r1 = float(input("digite a primeira reta: "))
r2 = float(input("digite a segunda reta: "))
r3 = float(input("digite a terceira reta: "))

if (r1 + r2 > r3) and (r1 + r3 > r2) and (r2 + r3 > r1):
    print("os segmentos acima PODEM FORMAR triângulo")
else:
    print("os segmentos acima NÃO PODEM FORMAR triângulo")
